let User,CustomErrorHandler;_fdc‍.x([["default",()=>_fdc‍.o]]);_fdc‍.w("../../models",[["User",["User"],function(v){User=v}]]);_fdc‍.w("../../services/CustomErrorHandler",[["default",["CustomErrorHandler"],function(v){CustomErrorHandler=v}]]);


const userController = {
  async me(req, res, next) {
    try {
      const user = await User.findOne({_id: req.user._id}).select(
        "-password -updatedAt -__v "
      );
      if (!user) {
        return next(CustomErrorHandler.notFound());
      }
      res.json(user);
    } catch (error) {
      return next(error);
    }
  },
};

_fdc‍.d(userController);
